expyfun/_experiment_controller.py

```diff
@@ -79,7 +79,7 @@ class ExperimentController(object):
     force_quit : list
         Keyboard key(s) to utilize as an experiment force-quit button. Can be
         a zero-element list for no force quit support. If None, defaults to
-        ``['lctrl', 'rctrl']``.  Using ['escape'] is not recommended due to
+        ``['lctrl', 'rctrl']``.  Using ``['escape']`` is not recommended due to
```

Comment:
So are we using single or double backticks nowadays?  Some of the below lines are using single...